pkg/storage/storage.go

```diff
@@ -88,6 +88,69 @@ type TupleBackend interface {
 	RelationshipTupleWriter
 }
 
+// ReadRelationshipTupleOpts is a functional option type that allows us to pass parameters.
+type ReadRelationshipTupleOpts struct {
```

Comment:
@smeadows-okta this struct leaks DynamoDB specific details. ConsistentRead and ScanIndexForward should not be part of our storage interface. They are specific to implementation details of DynamoDB.

Reply:
Corrected.